ferret/explainers/lime.py

```diff
@@ -87,27 +80,42 @@ def fn_prediction_token_ids(token_ids_sentences: List[str]):
 
             return logits.softmax(-1).detach().cpu().numpy()
 
-        # Same word has a different relevance according to its position
-        lime_explainer = LimeTextExplainer(bow=False, **self.init_args)
+        #added run_lime_explainer function to facilitate the handling of multiple-choice task
+        def run_lime_explainer(token_ids, target_pos_idx, num_samples, lime_args):
```

Comment:
Can you add a short short short docstring to the method and type hints on the parameters?

Reply:
Done!